packages/web/src/app/doc-pages/brand/illustrations-doc/illustrations-exhibit/illustrations-exhibit-details/illustrations-exhibit-details.component.html

```diff
@@ -0,0 +1,54 @@
+<div class="illustration">
+  <app-illustrations-generator [illustrationName]="selectedIllustration()" [color]="colorValue()" />
+</div>
+
+<div class="info">
+  <div class="installation-guide">
+    <div>
+      <span class="step-title e-text-md-strong e-mb-8"
+        ><i class="e-icon e-icon--step_1 e-icon--sm"></i> Import</span
+      >
+      <app-copy [message]="importStatement">
+        <code class="e-flex e-flex-direction-row e-align-items-center e-gap-8">
+          {{ importString }}
+        </code>
+      </app-copy>
+    </div>
+    <div>
+      <span class="step-title e-text-md-strong e-mb-8"
+        ><i class="e-icon e-icon--step_2 e-icon--sm"></i>HTML</span
+      >
+      <app-copy [message]="usageCopyString">
+        <code class="e-flex e-flex-direction-row e-align-items-center e-gap-8">
+          {{ usageString }}
+        </code>
+      </app-copy>
+    </div>
+  </div>
+  <div class="download">
+    <span class="step-title e-text-md-strong e-mb-16">Download</span>
```

Comment:
We should translate this to  "last ned" for our less technical Norwegian users here since they cant right click and press "large som...." or alike